src/components/Context/Context.tsx

```diff
@@ -1,87 +1,89 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable sonarjs/no-duplicate-string */
```

Comment:
You should fix code and remove these lines here and in other places where you used them - every line with disabling, except this with `@ts` prefix

Reply:
There are some issues that I am facing, @magicmatatjahu 

###  `/* eslint-disable github/array-foreach */`

I was getting this error because of `foreach` but I am facing a error, where when I try to convert the `foreach` to `for .. of` . The  `errorsInfo.push(`${e.title} ${e.location.startLine}:${e.location.startColumn}`);` is throwing error that `Type number is not assignable to type ValidationResponse` but I am not passing any number. 
<details>
<summary>error screenshot</summary>

![Screenshot (8)](https://user-images.githubusercontent.com/41781438/126599003-97cdb01f-d99c-4eb8-b41b-cf3260da708f.png)

</details>

### `/* eslint-disable security/detect-non-literal-fs-filename */`

This comes from `security` plugin, found these but I guess there isn't any workaround 
https://stackoverflow.com/questions/63262683/how-to-fix-found-fs-readfile-with-non-literal-argument-at-index-0
I guess the only way to deal with it is by disabling the rule. 

### `/* eslint-disable security/detect-object-injection */` 
not sure how to deal with this,

Reply:
@Souvikns 

Ok, you can use the `/* eslint-disable github/array-foreach */` only for this single case.

Please commit changes what you have, and I will look at it from the master branch :)